ui/public/locales/pl.json

```diff
@@ -623,8 +623,8 @@
 "label.diskiopswriterate": "Disk Write Rate (IOPS)",
 "label.diskioread": "Disk Read (IO)",
 "label.diskiowrite": "Disk Write (IO)",
-"label.diskkbsread": "Disk Read (Bytes)",
-"label.diskkbswrite": "Disk Write (Bytes)",
+"label.diskkbsread": "Disk Read",
+"label.diskkbswrite": "Disk Write",
```

Comment:
should these contain a `(KiB)` as well?

Reply:
They shouldn't, because these labels are used as column titles in instances tab when the metrics toggle is on. And to maintain the same pattern with the other columns, the units of measurements should be alongside the values, not the title.

Reply:
@JoaoJandre but they are included in the other languages for `label.diskkbsread` and `label.diskkbswrite`, and not in the polish json. Why is that?